src/esockd_acceptor.erl

```diff
@@ -44,22 +44,22 @@
           sockname     :: {inet:ip_address(), inet:port_number()},
```

Comment:
tips: Don't forget the `appup.src` If it is not to upgade the major, minor version number